pages/inventories/keycaps/[id].js

```diff
@@ -0,0 +1,350 @@
+          <HeaderImage>
+            <Image
+              src={keycaps.render_pics[0]}
+              alt={keycaps.name}
+              layout="fill"
+              objectFit="cover"
+            />
+          </HeaderImage>
+        )}
+      </HeaderSection>
+
+      <BoxContainer>
+        <li>
+          <strong>Manufacturer:</strong> {keycaps.keycapstype}
+        </li>
+        <li>
+          <strong>Profile:</strong> {keycaps.profile}
+        </li>
+        <li>
+          <strong>Designer:</strong> {keycaps.designer}
+        </li>
+        <li>
+          <strong>Geekhack Thread:</strong>{" "}
+          <ExternalLink href={keycaps.link} target="_blank">
+            Visit Geekhack
+          </ExternalLink>
+        </li>
+      </BoxContainer>
+
+      <h3>Your Kits</h3>
+      {kitsAvailable.length > 0 ? (
+        <GridContainer>
+          {kitsAvailable.map((kit) => (
+            <KitCard key={kit.name}>
+              {kit.pic && (
+                <Image
+                  src={kit.pic}
+                  alt={kit.name}
+                  width={100}
+                  height={100}
+                  objectFit="cover"
+                />
+              )}
+              <p>{kit.name}</p>
+            </KitCard>
+          ))}
+        </GridContainer>
+      ) : (
+        <p>No kits available for this keycap set.</p>
+      )}
+
+      <h3>Choose 3 Colors</h3>
+      <StyledInput
+        as="select"
+        onChange={handleColorSelect}
+        value=""
+        maxWidth="400px"
+      >
+        <option value="" disabled>
+          -- Choose up to 3 colors --
+        </option>
+        {colorOptions.map((color) => (
+          <option key={color.name} value={color.name}>
+            {color.name}
+            {color.emoji}
+          </option>
+        ))}
+      </StyledInput>
+
+      <GridContainer flexMode padding="10px" border bgColor="#f9f9f9">
+        {selectedColors.length > 0
+          ? selectedColors.map((color) => {
+              const colorData = colorOptions.find(
+                (option) => option.name === color
+              );
+              return (
+                <SelectedColor key={color} bgColor={colorData?.name}>
+                  {colorData?.emoji} {color}
+                  <RemoveColorButton onClick={() => handleRemoveColor(color)}>
+                    x
+                  </RemoveColorButton>
+                </SelectedColor>
+              );
+            })
+          : "No colors selected"}
+      </GridContainer>
+
+      <h3>Notes</h3>
+      <StyledInput
+        type="text"
+        maxLength={100}
+        placeholder="Write a note (max 100 chars)..."
+        value={newNote}
+        onChange={(event) => setNewNote(event.target.value)}
+      />
+      <BaseButton bgColor="#28a745" onClick={handleAddNote}>
+        Submit Note
+      </BaseButton>
+
+      {notes.length > 0 ? (
+        notes.map((note, index) => (
+          <BoxContainer bgColor="#f9f9f9" key={index}>
```

Comment:
You chose the BoxContainer to be an 'ul' element. This results to having a list of `<ul>` elements.
Wrap the mapping inside an `<ul>` element and make the BoxContainer to be a `<li>` element.